modules/mapbox/src/deck-utils.ts

```diff
@@ -30,30 +30,23 @@ export function getDeckInstance({
     return map.__deck;
   }
 
+  // Only initialize certain props once per context
   const customRender = deck?.props._customRender;
 
-  const deckProps: DeckProps = {
-    useDevicePixels: true,
-    _customRender: () => {
-      map.triggerRepaint();
-      // customRender may be subscribed by DeckGL React component to update child props
-      // make sure it is still called
-      // Hack - do not pass a redraw reason here to prevent the React component from clearing the context
-      // Rerender will be triggered by MapboxLayer's render()
-      customRender?.('');
-    },
-    // TODO: import these defaults from a single source of truth
-    parameters: {
-      depthMask: true,
-      depthTest: true,
-      blend: true,
-      blendFunc: [gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA, gl.ONE, gl.ONE_MINUS_SRC_ALPHA],
-      polygonOffsetFill: true,
-      depthFunc: gl.LEQUAL,
-      blendEquation: gl.FUNC_ADD
-    },
-    views: (deck && deck.props.views) || [new MapView({id: 'mapbox'})]
-  };
+  const deckProps = getDeckProps({
+    gl,
+    currProps: {
+      ...deck?.props,
+      _customRender: () => {
```

Comment:
I choose to not reapply the `_customRender` prop since it would stack infinitely. Seems to only need to be set once anyways.